tasks/utilities/submission/task_mercury_file_wrangling.wdl

```diff
@@ -251,18 +251,18 @@ task sm_metadata_wrangling { # the sm stands for supermassive
         print("DEBUG: creating genbank metadata table...")
         genbank_metadata = table[genbank_required].copy()
 
-        update_country = false
+        update_country = "false"
         for column in genbank_optional:
           if column in table.columns:
             genbank_metadata[column] = table[column]
             if column == "state":
-              update_country = true
+              update_country = "true"
```

Comment:
Could this also be addressed by making them proper python booleans, i.e. `True` and `False`?